### PR TITLE
Laser tracker LUTs notebooks

### PR DESCRIPTION
**Pull request by gmegh**

### CODE REVIEW

**Comment by b1quint** [comment]
Hey @gmegh! I see that your notebooks have outputs. Before anything else, please restart your notebooks and clear the outputs. Then, I will ask you to squash all your commits to remove the outputs from the timeline. Otherwise, they will be trapped in there and it will be super hard to remove them. 

I will ask you to run `pre-commit install` in your local repo copy. This will enable the pre-commit hook that checks for outputs in notebooks.

**Comment by b1quint** [comment]
Do we have a TN for the lasertracker tests? If not, do you think we will ever have? If so, maybe we put these notebooks inside the TN repo instead of here? I am thinking of doing the same for other notebooks/TNs.

**Comment by b1quint** [comment]
Make sure that all the notebooks have the following code in the first cell: 
```
%load_ext lab_black
%load_ext autoreload
%autoreload 2
```

File: notebooks/tel_and_site/subsys_req_ver/lasertracker/SITCOM_1215-LUTAzimuth.ipynb
Comment (b1quint) [inline notebooks/tel_and_site/subsys_req_ver/lasertracker/SITCOM_1215-LUTAzimuth.ipynb:1]: The plots in this file do not show up. I added `%matplotlib inline` to each plot cell and that helped bringing the plots up.

**Comment by b1quint** [comment]
The notebooks seem self-explanatory but, if you can add a paragraph or a few sentences explaining what are you looking at it would be super useful. Maybe even a "spoiler" saying what we will see and what is the preliminary conclusion. Not needed but, again, it would be useful for the record.

**Comment by gmegh** [comment]
Alright, I think I have addressed your comments, but please let me know if there is anything missing. We do have a TN which is in PR and I have to fix it: https://github.com/lsst-sitcom/sitcomtn-089/pull/3

But I think we would actually like to put these notebooks and tools into https://github.com/lsst-sitcom/aos_analysis/ which I recently created to have all the AOS scripting. Josh and I thought it might be easier to have it together, to access it? 

Should I create the PR there instead of here in that case?

**Review by b1quint — APPROVED** [review]
It looks good now. Thank you. If you want to keep things simple, merge these notebooks here until we find a better place for them. It is important that they are available for other people to use.

**Pull request merged**